vue/src/mapstyle/annotationStyles.ts

```diff
@@ -101,26 +101,135 @@ const styles: AnnotationStyle = {
   },
 };
 
+export type scoringColorsKeys = keyof typeof scoringColors;
+const scoringColors = {
+  2: {
+    color: 'lime',
+    title: 'True Positive (GT)',
+    hex: '#2AFF00',
+    description:'Positive site successfully detected (GT)',
+    simple: true,
+    detailed: true,
+    id: 2,
+  },
+  3: {
+    color: 'black',
+    hex: '#000000',
+    title: 'False Negative (GT)',
+    description:'Positive site not detected (GT)',
+    simple: true,
+    detailed: true,
+    id: 3,
+  },
+  5: {
+    color: 'red',
+    hex: '#FF0000',
+    title: 'False Positive (GT)',
+    description:'Negative annotation detected (GT)',
+    simple: true,
+    detailed: true,
+    id: 5,
+  },
+  7: {
+    color: 'thistle',
+    hex: '#CAA7FF',
+    title: 'Positive Site - Unbounded (GT)',
+    description:'Positive site - incomplete activity (GT)',
+    simple: false,
+    detailed: true,
+    id: 7,
+  },
+  10: {
+    color: 'salmon',
+    hex: '#FF8686',
+    title: 'Ignore Area (GT)',
+    description:'Ignore Area (GT)',
+    simple: false,
+    detailed: true,
+    id: 10,
+  },
+  14: {
+    color: 'lime',
+    title: 'True Positive (GT)',
+    hex: '#2AFF00',
+    description:'Positive site successfully detected (GT)',
+    simple: true,
+    detailed: true,
+    id: 14,
+  },
+  15: {
+    color: 'black',
+    hex: '#000000',
+    title: 'False Negative (GT)',
+    description:'Positive site not detected (GT)',
+    simple: true,
+    detailed: true,
+    id: 15,
+  },
+  17: {
+    hex: '#FF0000',
+    title: 'False Positive (GT)',
+    description:'Negative annotation detected (GT)',
+    simple: true,
+    detailed: true,
+    id: 17,
+  },
+  19: {
+    color: 'salmon',
+    hex: '#FF8686',
+    title: 'Ignore Area (GT)',
+    description:'Ignore Area (GT)',
+    simple: false,
+    detailed: true,
+    id: 19,
+  },
+  22: {
+    color: 'magenta',
+    hex: '#D53FFF',
+    title: 'False Positive (Proposal)',
+    description:'False Positive (Proposal)',
+    simple: true,
+    detailed: true,
+    id: 22,
+  },
+  23: {
+    color: 'orange',
+    title: 'Partial Positive (Proposal)',
+    hex: '#FF9033',
+    description:'Partial positive - Associates with both a positive site and a negative site (Proposal)',
+    simple: false,
+    detailed: true,
+    id: 23,
+  },
+  24: {
+    color: 'aquamarine',
+    hex: '#33FFEF',
+    title: 'True Positive (Proposal)',
+    description:'True Positive (Proposal)',
+    simple: false,
+    detailed: true,
+    id: 24,
+  },
+  25: {
+    color: 'magenta',
+    hex: '#D53FFF',
+    title: 'False Positive (Proposal)',
+    description:'False Positive (Proposal)',
+    simple: false,
+    detailed: true,
+    id: 25,
+  },
+}
```

Comment:
Would we want to add an endpoint for fetching these from the server instead of hardcoding them here? (In the future, not saying this should be done on this PR)